packages/webdriver-utils/src/main.js

```diff
@@ -0,0 +1,12 @@
+import ProviderResolver from './providers/providerResolver.js';
+import utils from '@percy/sdk-utils';
```

Comment:
```suggestion
import logger from '@percy/sdk-utils';
```

you can directly do this